src/domain/community/virtualContributor/vcProfilePage/SpaceHorizontalCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from 'react';
+import BadgeCardView from '../../../../core/ui/list/BadgeCardView';
+import Avatar from '../../../../core/ui/avatar/Avatar';
+import { BlockSectionTitle } from '../../../../core/ui/typography';
+import { useTranslation } from 'react-i18next';
+import { theme } from '../../../../core/ui/themes/default/Theme';
+import RouterLink from '../../../../core/ui/link/RouterLink';
+import defaultJourneyCardBanner from '../../../../domain/journey/defaultVisuals/Card.jpg';
+
+// TODO: add cardBanner if we want support of Spaces as BOK
+export interface BokProps {
+  bokProfile:
+    | {
+        avatar?: {
+          uri: string;
+        };
+        displayName: string;
+        tagline: string;
+        url: string;
+      }
+    | undefined;
+  showDefaults?: boolean;
+}
+
+const DEFULT_BOK = {
```

Comment:
I think this should go somewhere else, it's own file 
and maybe be translatable, that's not that easy...

Reply:
we're keeping it for the MVP and the support of legacy Virtual contributors